kong/credentials_service.go

```diff
@@ -11,15 +11,15 @@ import (
 type abstractCredentialService interface {
 	// Create creates a credential in Kong of type credType.
 	Create(ctx context.Context, credType string, consumerUsernameOrID *string,
-		credential interface{}) (json.RawMessage, error)
-	// Get fetches a credential of credType with consumerIdentifier and credIdentifier from Kong.
+		credential interface{}, skipHash bool) (json.RawMessage, error)
```

Comment:
Could we define types such as `createCredentialOptions` to make the interface clearer and more extensible?